tests/DisplayAllArchivedRecordsTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once 'src/dispayAllArchivedRecordsFunction.php';
+
+class DisplayAllArchivedRecordsTest extends TestCase
+{
+    public function test_success_displayAllRecords()
+    {
+        //inputs
+        $records = [
+            (object)[
```

Comment:
Rather than being a generic object (this syntax is creating an assoc array, and then forcing it to become php's generic object) you should be intantiating one of your entities